bundles/com.espressif.idf.terminal.connector.serial/src/com/espressif/idf/terminal/connector/serial/launcher/GDBStubDebuggerLauncher.java

```diff
@@ -0,0 +1,251 @@
+package com.espressif.idf.terminal.connector.serial.launcher;
+
+import java.io.File;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.debug.internal.core.LaunchConfiguration;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.w3c.dom.Attr;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import com.espressif.idf.core.util.IDFUtil;
+import com.espressif.idf.core.util.SDKConfigJsonReader;
+
+public class GDBStubDebuggerLauncher
+{
```

Comment:
Could you mention the original class if this is derived from some other source class in the class comments section so that in the future it will be easy to adopt new changes